app/models/alchemy/content/factory.rb

```diff
@@ -122,19 +122,26 @@ def descriptions
       # Returns a normalized Essence type
       #
       # Adds Alchemy module name in front of given essence type
+      # unless there is a Class with the specified name that is an essence.
       #
       # @param [String]
       #   the essence type to normalize
       #
       def normalize_essence_type(essence_type)
         essence_type = essence_type.classify
-        if essence_type.match(/\AAlchemy::/)
-          essence_type
-        else
-          essence_type.gsub!(/\AEssence/, 'Alchemy::Essence')
-        end
+        return essence_type if is_an_essence?(essence_type)
+
+        "Alchemy::" + essence_type
```

Comment:
Sorry, one last change ;)

We prefer interpolated string over concatenated strings. 

"Alchemy::#{essence_type}"

Please 😊